db/migrate/20200731213827_add_acuant_result_to_proofing_costs.rb

```diff
@@ -1,5 +0,0 @@
-class AddAcuantResultToProofingCosts < ActiveRecord::Migration[5.2]
```

Comment:
Normally I think it's bad to delete migrations? But since this failed to run in production, I think it's safe. Some local devs might be messed up if they already have the column